packages/adapter-ccxt/.env.example

```diff
@@ -0,0 +1,4 @@
+# Used for local testing purposes only
+# Test credentials can be obtained from https://testnet.binance.vision/
```

Comment:
We could remove the example strings here or move them into a comment. Because then we can check in the code if an env var is 

a) set (not null) and
b) of the right type

if we did this:

```sh
# Test credentials can be obtained from https://testnet.binance.vision/
# i.e. ABCDEF12345
BINANCE_API_KEY=
```

then in the code we could more easily check if our env vars are correct

```ts
if (!process.env.BINANCE_API_KEY) // would now work vs before
if (typeof process.env.BINANCE_API_KEY !== 'string') // would now work vs before
```



```suggestion
# Test credentials can be obtained from https://testnet.binance.vision/
# i.e. aaaaaaaaaaaaaaa11aaa1aaaaa7aaaaaaaaaaaaaaaaaaaaaaa1aaa1aa11aaaaa
BINANCE_API_KEY=
BINANCE_API_SECRET=
```